changelog.d/435.change.rst

```diff
@@ -0,0 +1 @@
+Allow users to customize behaviour of ``eq`` and ``order`` by passing in a ``callable``.
```

Comment:
American English please :)

Also we don't write our Changelog in git-commit message tense.  More like "It's now possible to customize…"

Callable doesn't have to be wrapped in backticks here – it can just stand as a concept.

Reply:
Fixed.